### PR TITLE
Remove Hub code

### PR DESCRIPTION
**Pull request by tieujason330**
### What changes are proposed in this pull request?
See title.

### Does this PR introduce any user facing changes?
No

### CODE REVIEW

File: integration/dataproc/alluxio-dataproc.sh [hunk 1]
```diff
@@ -139,7 +139,7 @@ expose_alluxio_client_jar() {
 
 configure_alluxio_systemd_services() {
   if [[ "${ROLE}" == "Master" ]]; then
-    # The master role runs 3 daemons: AlluxioMaster and AlluxioJobMaster and HubManager
+    # The master role runs 3 daemons: AlluxioMaster and AlluxioJobMaster
```
Comment (Xenorith) [inline integration/dataproc/alluxio-dataproc.sh:142]: 2 daemons?

**Review by Xenorith — APPROVED** [review]

File: integration/dataproc/alluxio-dataproc.sh (hunk 1 above)
Comment (tieujason330) [reply integration/dataproc/alluxio-dataproc.sh:142]: updated

**Comment by tieujason330** [comment]
alluxio-bot, merge this please

**Comment by tieujason330** [comment]
alluxio-bot, merge this please

**Comment by maobaolong** [comment]
@tieujason330 Hi, why we remove the hub related codes?

**Pull request merged**